Packs/CommonScripts/Scripts/IsIPPrivate/IsIPPrivate.py

```diff
@@ -0,0 +1,111 @@
+from ipaddress import IPv4Address, IPv4Network
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+DELIMITER = ","
+
+
+def is_ip_internal(ip: str, ranges: List[str]) -> bool:
+    try:
+        return any((IPv4Address(ip) in IPv4Network(cidr.split(DELIMITER)[0] if DELIMITER in cidr else cidr) for cidr in ranges))
+    except ValueError:
+        demisto.info("One or more IP ranges or IPs are invalid. Please make sure the list is in the correct structure.")
+        return True
+
+
+def get_ip_tag(ip: str, ranges: List[str]) -> str:
+    for cidr in ranges:
+        tag = None
+        if DELIMITER in cidr:
+            cidr, tag = cidr.split(DELIMITER)[0], cidr.split(DELIMITER)[1].strip()
+        try:
+            if IPv4Address(ip) in IPv4Network(cidr):
+                return tag or ''
+        except ValueError:
+            if ip == cidr:
+                return tag or ''
+    return ''
+
+
+def main():
+    args = demisto.args()
+    ranges_list_name = args.get("PrivateIPsListName", "PrivateIPs")
+    ip_addresses_to_check = argToList(args.get("IPAddresses"))
+
+    # Get the list of private IP ranges from the XSOAR list:
+    private_ranges = demisto.executeCommand("getList", {"listName": ranges_list_name})[0]['Contents']
```

Comment:
it's ok, but FYI for next time, we prefer using `execute_command()` from CSP.

Reply:
ok, I tried to replace with `execute_command` but got ` TypeError: string indices must be integers`. I'm leaving it with demisto.executeCommand for now